src/gui/folderman.cpp

```diff
@@ -594,7 +597,8 @@ Folder *FolderMan::folder(const QString &alias)
 
 void FolderMan::scheduleAllFolders()
 {
-    for (Folder *f : _folderMap.values()) {
+    const auto folderMapValues = _folderMap.values();
+    for (Folder *f : folderMapValues) {
```

Comment:
might be out of scope of this PR, but `auto`

Reply:
also, I don't think single letter variables are ever a good idea (though that might be slightly out of scope of this PR too)